gandalf.js

```diff
@@ -1,90 +1,93 @@
-const isPalindrome = word => word === word.split("").reverse().join("");
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?